Worksheet.md

```diff
@@ -134,15 +136,22 @@ This takes the **output** of `grep` and _pipes_ it to the **input** of the `wc`
 
 ### System administration
 
-There are many tools you can use in Linux to keep on top of your computer.
+There are many tools you can use in Linux to keep on top of your computer, as well as some useful shortcuts and tricks. 
 
-The first tool we'll look at is `man` - these are "manual pages" (also referred to as manpages), and they provide explanation of the commands in Linux and all of the different things you can do with them. Once you're looking at a manpage, you can use the arrow keys on your keyboard to scroll through the document, and press the `Q` key on your keyboard to exit.
+- `Ctrl + C` is a keyboard interrupt, and stops whatever process is being run on your terminal window. 
+- Adding `&` to the end of a command in a terminal will make it run in the background, letting you do other things while it's running. (You can also use `command1 && command2` if you want to run two commands one after the other). 
+- Copying and pasting in terminal windows uses `Ctrl + Shift + C/V` rather than just `Ctrl + C/V`. 
```

Comment:
Not necessarily - this depends on your terminal emulator, and doesn't cover Macs.